src/types.js

```diff
@@ -51,6 +51,9 @@ export type ButtonThemeConfig = {
     large?: Stylesheet
   }
 };
+export type CalloutThemeConfig = {
+  base?: Stylesheet
```

Comment:
We will need to add `Header`, `Content`, `Footer`, `Title` and `Icon` as well. And also edit the `themes.mdx` to include `CalloutThemeConfig`.